jablib/src/main/java/org/jabref/logic/util/io/FileUtil.java

```diff
@@ -589,4 +593,47 @@ public static String shortenFileName(String fileName, Integer maxLength) {
     public static boolean isCharLegal(char c) {
         return Arrays.binarySearch(ILLEGAL_CHARS, c) < 0;
     }
+
+    /// Converts a Cygwin-style file path to a Windows-style path if the operating system is Windows.
+    ///
+    /// Supported formats:
+    /// - /cygdrive/c/Users/... → C:\Users\...
+    /// - /mnt/c/Users/...      → C:\Users\...
+    /// - /c/Users/...          → C:\Users\...
+    ///
+    /// @param filePath the input file path
+    /// @return the converted path if running on Windows and path is in Cygwin format; otherwise, returns the original path
+    public static Path convertCygwinPathToWindows(String filePath) {
```

Comment:
The method also works on linux - proposal for another name:

```suggestion
    public static Path convertStringToPathRespecingCygwin(String filePath) {
```